docs/DevelopmentPlan/DevelopmentPlan.md

```diff
@@ -86,13 +92,26 @@ of the individuals will likely change throughout the year.
 
 # Project Decomposition and Scheduling
 
--   How will you be using GitHub projects?
-
--   Include a link to your GitHub project
-
-[\[How will the project be scheduled? This is the big picture schedule,
-not details. You will need to reproduce information that is in the
-course outline for deadlines. ---SS\]]{style="color: blue"}
+-   We will be using Github Projects to decompose our milestones. Each Milestone will be divided
+into individual tasks which will then be assigned to different team members.
+- Each document will be split into sections, and assigned individually or to groups as needed. 
+- The proof of concept, revision 0 and revision 1 will be split feature-wise and assigned to individuals or groups as necessary.
+- Time will be alloted for verification, validation and review after the completion of each milestone and before the due date.
+
+-   [Our GitHub Project](https://github.com/users/Spitgranger/projects/2/views/1)
+
+**Milestone** | **Schedule** | 
```

Comment:
hey I left this same comment in the google doc, but as per this [message](https://teams.microsoft.com/l/message/19:7552078e4c1545be8b3892290739e3f3@thread.tacv2/1726799669900?tenantId=44376307-b429-42ad-8c25-28cd496f4772&groupId=96bd54ff-6781-4b7d-a320-a366b997f977&parentMessageId=1726789091786&teamName=SE%204G06%202024-2025&channelName=01.%20Problem%20Statement%2C%20Goals%20and%20Dev%20Plan&createdTime=1726799669900) it doesn't look like the prof wants a schedule like this. Instead, he wants an explanation of how we will handler decomposing larger tasks into smaller ones and assigning out work

Reply:
See https://gitlab.cas.mcmaster.ca/courses/capstone/-/blob/main/Lectures/L02b_POCAndDevPlan/POCAndDevPlan.pdf?ref_type=heads side 14